spec/mongoid/criteria/queryable/selector_spec.rb

```diff
@@ -71,6 +71,28 @@
           })
         end
       end
+
+      context "when merging in a new symbol $nin" do
+
+        let(:other) do
+          { "field" => { "$nin" => ["bar"] } }
+        end
+
+        before do
+          selector["field"] = { :$nin => ["foo"] }
+          selector.merge!(other)
+        end
+
+        it "combines the two $nin queries into one" do
+          expect(selector).to eq({
+            "field" => { "$nin" => ["foo", "bar"] }
+          })
+        end
+
+        after do
+          selector["field"] = initial
```

Comment:
This should not be necessary, `selector` is a `let`, which gets (re-)created for every test